modules/gdscript/gdscript.cpp

```diff
@@ -949,7 +949,12 @@ bool GDScriptInstance::set(const StringName &p_name, const Variant &p_value) {
 			if (E->get().setter) {
 				const Variant *val = &p_value;
 				Variant::CallError err;
-				call(E->get().setter, &val, 1, err);
+				if (E->get().setter_no_set) {
+					call(E->get().setter, NULL, 0, err);
+					members.write[E->get().index] = p_value;
```

Comment:
Shouldn't you write to the variable before calling the "setter"?

Reply:
I've written to the variable in the latest force push, thanks!